eng/Tools.props

```diff
@@ -5,6 +5,7 @@
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
       https://dotnet.myget.org/F/sourcelink/api/v3/index.json;
+      https://www.myget.org/F/vihofer-staging/api/v3/index.json;
```

Comment:
Will you remove this temporary feed later?

Reply:
yes, before the PR goes in.